PlexTraktSync rejects command

### App Name

PlexTraktSync

### Operating System

TrueNAS SCALE 23.10.0'or 23.10.0.1

### App Version

0.27.11

### Application Events

```Shell
2023-12-07 16:07:12
Add eth0 [172.16.9.75/16] from ix-net
2023-12-07 16:07:12
Container image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" already present on machine
2023-12-07 16:07:12
Created container plextraktsync
2023-12-07 16:07:12
Started container plextraktsync
2023-12-07 16:07:11
Created pod: plextraktsync-cron-28366080-dckzq
2023-12-07 16:07:11
Successfully assigned ix-plextraktsync/plextraktsync-cron-28366080-dckzq to ix-truenas
2023-12-07 16:06:09
Add eth0 [172.16.9.74/16] from ix-net
2023-12-07 16:06:09
Container image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" already present on machine
2023-12-07 16:06:09
Created container plextraktsync
2023-12-07 16:06:09
Started container plextraktsync
2023-12-07 16:06:08
Created pod: plextraktsync-cron-28366080-q29cx
2023-12-07 16:06:08
Successfully assigned ix-plextraktsync/plextraktsync-cron-28366080-q29cx to ix-truenas
2023-12-07 16:05:02
Created container plextraktsync
2023-12-07 16:05:02
Started container plextraktsync
2023-12-07 16:05:01
Created pod: plextraktsync-cron-28366080-g5js7
2023-12-07 16:05:01
Successfully assigned ix-plextraktsync/plextraktsync-cron-28366080-g5js7 to ix-truenas
2023-12-07 16:05:01
Add eth0 [172.16.9.73/16] from ix-net
2023-12-07 16:05:01
Container image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" already present on machine
2023-12-07 16:04:13
Created container plextraktsync
2023-12-07 16:04:13
Started container plextraktsync
2023-12-07 16:04:12
Created pod: plextraktsync-cron-28366080-52x5g
2023-12-07 16:04:12
Successfully assigned ix-plextraktsync/plextraktsync-cron-28366080-52x5g to ix-truenas
2023-12-07 16:04:12
Add eth0 [172.16.9.72/16] from ix-net
2023-12-07 16:04:12
Container image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" already present on machine
2023-12-07 16:03:39
Created container plextraktsync
2023-12-07 16:03:39
Started container plextraktsync
2023-12-07 16:03:38
Add eth0 [172.16.9.71/16] from ix-net
2023-12-07 16:03:38
Container image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" already present on machine
2023-12-07 16:03:37
Scaled up replica set plextraktsync-5b96b58f74 to 1
2023-12-07 16:03:37
Created pod: plextraktsync-5b96b58f74-v9dnq
2023-12-07 16:03:37
Successfully assigned ix-plextraktsync/plextraktsync-5b96b58f74-v9dnq to ix-truenas
2023-12-07 16:03:28
Created container plextraktsync
2023-12-07 16:03:28
Started container plextraktsync
2023-12-07 16:03:27
Add eth0 [172.16.9.70/16] from ix-net
2023-12-07 16:03:27
Container image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" already present on machine
2023-12-07 16:03:26
Created job plextraktsync-cron-28366080
2023-12-07 16:03:26
Created pod: plextraktsync-cron-28366080-fd48b
2023-12-07 16:03:26
Successfully assigned ix-plextraktsync/plextraktsync-cron-28366080-fd48b to ix-truenas
2023-12-07 16:01:23
Scaled down replica set plextraktsync-6bdc587b69 to 0 from 1
2023-12-07 16:01:23
Deleted pod: plextraktsync-6bdc587b69-v47w6
2023-12-07 16:01:23
Stopping container plextraktsync
2023-12-07 15:44:40
Created container plextraktsync
2023-12-07 15:44:40
Started container plextraktsync
2023-12-07 15:44:39
Add eth0 [172.16.9.67/16] from ix-net
2023-12-07 15:44:39
Container image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" already present on machine
2023-12-07 15:44:38
Scaled up replica set plextraktsync-6bdc587b69 to 1
2023-12-07 15:44:38
Created pod: plextraktsync-6bdc587b69-v47w6
2023-12-07 15:44:38
Successfully assigned ix-plextraktsync/plextraktsync-6bdc587b69-v47w6 to ix-truenas
2023-12-07 15:43:30
Stopping container plextraktsync
2023-12-07 15:43:29
Created container plextraktsync
2023-12-07 15:43:29
Started container plextraktsync
2023-12-07 15:43:27
Successfully pulled image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994" in 7m51.13240845s (7m51.132412898s including waiting)
2023-12-07 15:38:44
Deleted pod: plextraktsync-5cb446f76c-j67c8
2023-12-07 15:38:40
Scaled down replica set plextraktsync-5cb446f76c to 0 from 1
2023-12-07 15:35:36
Pulling image "tccr.io/truecharts/plextraktsync:v0.27.11@sha256:74efbebce5bf060c4e7b6c9936e6c551944957a25f388c70e0d1254e516a0994"
2023-12-07 15:35:29
Add eth0 [172.16.9.66/16] from ix-net
2023-12-07 15:35:11
Unable to attach or mount volumes: unmounted volumes=[config], unattached volumes=[config devshm shared tmp varlogs varrun]: error processing PVC ix-plextraktsync/plextraktsync-config: PVC is not bound
2023-12-07 15:34:59
Successfully assigned ix-plextraktsync/plextraktsync-5cb446f76c-j67c8 to ix-truenas
2023-12-07 15:34:59
Unable to attach or mount volumes: unmounted volumes=[varrun config devshm shared tmp varlogs], unattached volumes=[varrun config devshm shared tmp varlogs]: error processing PVC ix-plextraktsync/plextraktsync-config: PVC is not bound
2023-12-07 15:34:54
0/1 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod..
2023-12-07 15:34:36
failed to provision volume with StorageClass "ix-storage-class-plextraktsync": rpc error: code = Aborted desc = volume pvc-9eec48d7-deb6-4ce3-8cea-a834dea81c88 request already pending
2023-12-07 15:34:33
failed to provision volume with StorageClass "ix-storage-class-plextraktsync": rpc error: code = DeadlineExceeded desc = context deadline exceeded
2023-12-07 15:34:23
External provisioner is provisioning volume for claim "ix-plextraktsync/plextraktsync-config"
2023-12-07 15:33:35
failed to provision volume with StorageClass "ix-storage-class-plextraktsync": rpc error: code = Internal desc = not able to provision the volume, nodes [ix-truenas], err : zfs: wait failed, not able to get the volume pvc-9eec48d7-deb6-4ce3-8cea-a834dea81c88 zfsvolumes.zfs.openebs.io "pvc-9eec48d7-deb6-4ce3-8cea-a834dea81c88" not found
2023-12-07 15:33:32
failed to provision volume with StorageClass "ix-storage-class-plextraktsync": rpc error: code = Aborted desc = volume pvc-9eec48d7-deb6-4ce3-8cea-a834dea81c88 request already pending
2023-12-07 15:33:31
External provisioner is provisioning volume for claim "ix-plextraktsync/plextraktsync-config"
2023-12-07 15:33:30
Scaled up replica set plextraktsync-5cb446f76c to 1
2023-12-07 15:33:30
failed to provision volume with StorageClass "ix-storage-class-plextraktsync": rpc error: code = DeadlineExceeded desc = context deadline exceeded
2023-12-07 15:33:27
Created pod: plextraktsync-5cb446f76c-j67c8
2023-12-07 15:33:27
0/1 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod..
2023-12-07 15:33:20
waiting for a volume to be created, either by external provisioner "zfs.csi.openebs.io" or manually created by system administrator
```


### Application Logs

```Shell
2023-12-07 16:03:28.505802+00:00Usage: python -m plextraktsync [OPTIONS] COMMAND [ARGS]...
2023-12-07 16:03:28.505836+00:00Try 'python -m plextraktsync --help' for help.
2023-12-07 16:03:28.505841+00:002023-12-07T16:03:28.505841219Z
2023-12-07 16:03:28.505848+00:00Error: No such command 'sync --sync=all'.
```


### Application Configuration

Schedule changed to: `0 */1 * * *`

I set it up manually using the command line first time, which I believe is the intended setup as there is no documentation

### Describe the bug

The scheduled sync task does not run with the error message:

Error: No such command 'sync --sync=all'

Provided by the cron pod



### To Reproduce

1. Perform initial container setup using pod shell (run the plextraktsync command without any arguments and follow the on-screen steps)
2. Scheduled cron tasks fail to run sync command.

### Expected Behavior

You receive the error "no such command" and cron containers are infinitely made

### Screenshots

N/A

### Additional Context

N/A

### I've read and agree with the following

- [X] I've checked all open and closed issues and my issue is not there.

--- a/charts/stable/plextraktsync/Chart.yaml
+++ b/charts/stable/plextraktsync/Chart.yaml
@@ -1,7 +1,7 @@
 kubeVersion: '>=1.24.0-0'
 apiVersion: v2
 name: plextraktsync
-version: 6.1.8
+version: 6.1.9
 appVersion: 0.27.15
 description: Two-way-sync between trakt.tv and Plex Media Server.
 home: https://truecharts.org/charts/incubator/plextraktsync

--- a/charts/stable/plextraktsync/ci/cron-values.yaml
+++ b/charts/stable/plextraktsync/ci/cron-values.yaml
@@ -1,0 +1,4 @@
+plextraktsync:
+  cron_enabled: true
+  task: ""
+  schedule: "*/1 * * * *"

--- a/charts/stable/plextraktsync/templates/_cronjob.tpl
+++ b/charts/stable/plextraktsync/templates/_cronjob.tpl
@@ -10,13 +10,12 @@ podSpec:
       enabled: true
       primary: true
       imageSelector: "image"
-      command:
-          - plextraktsync
-          {{- if  .Values.plextraktsync.task }}
-          - {{ .Values.plextraktsync.task }}
-          {{- else }}
-          - sync --sync=all
-          {{- end }}
+      args:
+        {{- if  .Values.plextraktsync.task }}
+        - {{ .Values.plextraktsync.task }}
+        {{- else }}
+        - sync --sync=all
+        {{- end }}
       probes:
         liveness:
           enabled: false

--- a/charts/stable/rss-bridge/Chart.yaml
+++ b/charts/stable/rss-bridge/Chart.yaml
@@ -1,7 +1,7 @@
 kubeVersion: ">=1.24.0-0"
 apiVersion: v2
 name: rss-bridge
-version: 8.1.5
+version: 8.1.6
 appVersion: "2023"
 description: RSS-Bridge is a PHP project capable of generating RSS and Atom feeds for websites that don't have one.
 home: https://truecharts.org/charts/stable/rss-bridge

--- a/charts/stable/rss-bridge/values.yaml
+++ b/charts/stable/rss-bridge/values.yaml
@@ -11,6 +11,8 @@ persistence:
   config:
     enabled: true
     mountPath: "/config"
+  varrun:
+    enabled: false
 portal:
   open:
     enabled: true

